src/firebase-namespace.ts

```diff
@@ -219,7 +219,7 @@ export class FirebaseNamespaceInternals {
     this.firebase_[serviceName] = serviceNamespace;
 
     return serviceNamespace;
-  }
+  } */
```

Comment:
Is it a good idea to remove this if we're going down the module route?

Reply:
Is it currently used by anything?

Reply:
No